api.planx.uk/modules/analytics/metabase/collection/types.ts

```diff
@@ -0,0 +1,49 @@
+import { z } from "zod";
+import type { ValidatedRequestHandler } from "../../../../shared/middleware/validate.js";
+
+type ApiResponse<T> = {
+  data?: T;
+  error?: string;
+};
+
+/** Interface for incoming request, in camelCase */
+export interface NewCollectionParams {
+  slug: string;
+  description?: string;
+  /** Optional; if the collection is a child of a parent, specify parent ID here */
+  parentId?: number;
+}
+
+/** Interface for request after transforming to snake case (Metabase takes snake while PlanX API takes camel) */
+export type MetabaseCollectionParams = Omit<NewCollectionParams, "slug"> & {
+  name: string;
+};
+
+/** Metbase collection ID for the the "Council" collection **/
```

Comment:
```suggestion
/** Metabase collection ID for the the "Council" collection **/
```

Might be worth just adding another line explaining why commented out as per the review thread with Daf :)